cohorts/functions.py

```diff
@@ -96,7 +96,7 @@ def neoantigen_count(row, cohort, filter_fn=None,
     return np.nan
 
 def expressed_missense_snv_count(row, cohort, **kwargs):
-    filter_fn = kwargs.pop("filter_fn")
+    filter_fn = kwargs.pop("filter_fn", None)
```

Comment:
(I forgot that `expressed_filter_fn` is the new `filter_fn`, which doesn't give `Cohort.filter_fn` a chance to be used; since by the time we check for the default, we think we already have a `filter_fn`: `expressed_filter_fn`.)